R/est.incidence.by.R

```diff
@@ -101,6 +101,27 @@ est.incidence.by <- function(
     return(to_return)
   }
 
+  if (!any(is.element(strata, pop_data %>% names()))) {
+    # search strata variable from pop_data
+    strata_var <-
+      grep(
+        x = pop_data %>% names(),
+        value = TRUE,
+        pattern = strata,
```

Comment:
what happens here if `strata` has multiple entries in it? for example, `strata = c("Country", "catchment")`? Please check this scenario and adjust this code if needed (probably, try to match each stratum variable separately).

Reply:
This is in order. It adds complexity to Issue #233